blog/_posts/2021-07-06-distributed-rdataframe.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "RDataFrame is going distributed!"
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
+A recently introduced Python package enables distributing ROOT RDataFrame workloads to a set of remote resources.
+This feature is available in experimental phase since the latest ROOT 6.24 release, allowing users to write and run
+their applications from within the same interface while steering the computations to an Apache Spark cluster.
+
+## In detail
+
+[RDataFrame](https://root.cern/doc/master/classROOT_1_1RDataFrame.html) is ROOT's high-level interface for data
+analysis and since its release in 6.14 has seen tons of usages in real world analysis use cases and other examples in
+the wild. Parallelism has always been a staple of its design with support for executing the event loop on all cores
```

Comment:
```suggestion
[RDataFrame](https://root.cern/doc/master/classROOT_1_1RDataFrame.html) is ROOT's high-level interface for data
analysis since ROOT v6.14. By now many real world analyses use is, and on top of that we see lots of non-analysis usage in
the wild. Parallelism has always been a staple of its design with support for executing the event loop on all cores
```